paddle/fluid/framework/ir/mkldnn/layer_norm_onednn_optimization_pass.cc

```diff
@@ -1,111 +0,0 @@
-// Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Better to describe why delete some pass in PR commits

Reply:
Before oneDNN v3.x, layernorm primitive use a single arguments to pass both scale and shit, so we need this pass to cat scales and shift together. But in oneDNN 3.x, layernorm primitive can use separated scales and shit, this pass is not needed any more.